WordPress/gutenberg
# Post excerpt: add text alignment to the placeholder

## What?

Adds text alignment controls to the block toolbar of the post excerpt placeholder.
Closes https://github.com/WordPress/gutenberg/issues/44324

## Why?

The placeholder should have the same styling options as the regular block.

## How?

Adds the alignment toolbar to the placeholder.

## Testing Instructions
In the site editor, open the template for single posts.
Add an excerpt block  to the template (or convert an existing post content block to a post excerpt).
Confirm that the placeholder is showing and that you can align the text to the left/center/right using the toolbar.
Save and view a post on the front. Confirm that the text aligns to the left/center/right correctly.

## Screenshots or screencast
